stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/WriteCharacters.java

```diff
@@ -61,29 +71,75 @@
     private static final int START_OFFSET_INDEX = 0;
 
     /**
-     * Writes characters to a given file.
+     * Will be the I/O event handler.
+     */
+    private EventManager eventManager = EventManager.getInstance();
+
+    private static final Logger log = LoggerFactory.getLogger(WriteCharacters.class);
+
+/*
+    private static EventResult readCharactersResponse(EventResult<Integer, EventContext> result) {
+        BStruct errorStruct;
+        EventContext eventContext = result.getContext();
+        Context context = eventContext.getContext();
+        Throwable error = eventContext.getError();
+        if (null != error) {
+            errorStruct = IOUtils.createError(context, error.getMessage());
+        }
+        Integer numberOfBytes = result.getResponse();
+        return result;
+    }
+*/
```

Comment:
Remove commented block?

Reply:
This needs to be uncommented upon the availability of the async functions